test/index.js

```diff
@@ -15,76 +15,78 @@ function eq(actual, expected) {
   assert.strictEqual (Z.equals (actual, expected), true);
 }
 
-function expectOutput(expectations) {
-  var idx = 0;
-  var unhook = intercept (function(text) {
-    (text.split ('\n')).forEach (function(line) {
+const expectOutput = expectations => {
+  let idx = 0;
+  const unhook = intercept (text => {
+    (text.split ('\n')).forEach (line => {
       if (idx >= expectations.length) {
         unhook ();
         throw new Error (
-          'More lines than expected were printed:\n\n' +
-          JSON.stringify (line)
+          `More lines than expected were printed:
+
+${JSON.stringify (line)}`
         );
       }
       if (expectations[idx].test (line) === false) {
         unhook ();
         throw new Error (
-          'A printed line did not meet expectations:\n\n' +
-          JSON.stringify (line) + '\n\n' +
-          'It does not match expression:\n\n' +
-          expectations[idx].toString ()
+          `A printed line did not meet expectations:
+
+${JSON.stringify (line)}
+
+It does not match expression:
+
+${expectations[idx]}`
```

Comment:
This indentation bothers me. How do you feel about this, Aldwin?

Reply:
It bothers me too. You could use the [`deindent`](https://www.npmjs.com/package/deindent) package, or sometimes I also just fall back on something like:

```js
    `A printed line did not meet expectations:\n\n${
      JSON.stringify (line)
    }\n\nIt does not match expression:\n\n${
      expectations[idx]
    }`
```